src/pages/iou/IOUCurrencySelection.js

```diff
@@ -61,7 +61,9 @@ class IOUCurrencySelection extends Component {
      */
     getSections() {
         const sections = [];
-
+        if (this.state.searchValue.trim() && !this.state.currencyData.length) {
+            return [];
+        }
```

Comment:
I think Rajat also meant to move this above the `const sections = [];`.

Reply:
changes added into latest commit with GPG verified :)